packages/app/src/components/Normal/FloatingActionButton/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { ReactNode, useCallback, useState } from "react"
+import { FC } from "react"
+import { enhance } from "src/utilities/essentials"
+import { FloatingActionButtonFallback } from "./fallback"
+import { x } from "@xstyled/styled-components"
+import { Minus } from "@styled-icons/boxicons-regular/Minus"
+import { Add } from "@styled-icons/material-sharp/Add"
+import { V } from "src/styles/styleFragments"
+import { PlaylistPlay } from "styled-icons/material-sharp"
+import { Settings } from "styled-icons/material-sharp"
+import { Information } from "styled-icons/ionicons-sharp"
+
+export type Menu = {
+  icon: ReactNode
+  menuType: MenuType
+}
+
+enum MenuType {
+  DEFAULT,
+  PLAYLIST,
+  SETTINGS,
+  INFO,
+}
```

Comment:
정적인 언어에서 ~type이란 단어는 이미 type을 정의하고 있기 때문에 되도록이면 빼는 것을 추천
```suggestion
enum FABMenus {
  DEFAULT,
  PLAYLIST,
  SETTINGS,
  INFO,
}
```

Reply:
enum 삭제 했습니다.